app/provider_details/provider_service.py

```diff
@@ -56,15 +53,16 @@ def get_provider(
 
         # This is a UUID (ProviderDetails primary key).
         provider_id = self._get_template_or_service_provider_id(notification)
-        logger.debug('notification = %s', notification)
-        logger.debug('provider_id = %s', provider_id)
+        current_app.logger.debug('notification = %s, provider_id = %s', notification.id, provider_id)
```

Comment:
Nit: This log message was obviously left for testing, but I would like to see log messages have a little more context to them than just an ID or two thrown out there. Like, where the log is coming from for example.